IOPool/Streamer/test/NewStreamInPadding_cfg.py

```diff
@@ -0,0 +1,23 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("TRANSFER")
+
+import FWCore.Framework.test.cmsExceptionsFatal_cff
+process.options = FWCore.Framework.test.cmsExceptionsFatal_cff.options
+
+process.load("FWCore.MessageLogger.MessageLogger_cfi")
+
+process.source = cms.Source("NewEventStreamFileReader",
+    fileNames = cms.untracked.vstring('file:teststreamfile_padding.dat')
+    #firstEvent = cms.untracked.uint64(10123456835)
+)
+
+process.a1 = cms.EDAnalyzer("StreamThingAnalyzer",
```

Comment:
Yes, in `RunSimple_NewStreamer.sh` there are checksum comparisons. I didn't add it for padded format, but definitely I should add it.
Or maybe it can be done in a nicer way such as writing / reading checksum file from StreamThing modules without scripting involved)..

Reply:
Improved it as I said.
Interesting is, if exception was being thrown from destructor of StreamThingAnalyzer, an abort was happening somewhere (see below).
Therefore I moved the checksum check to endJob, but maybe it's interesting to investigate independently.
```
#3  0x00007fe26cba733b in sig_dostack_then_abort () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_13_1_0_pre1/lib/slc7_amd64_gcc11/pluginFWCoreServicesPlugins.so
#4  <signal handler called>
#5  0x00007fe272329387 in raise () from /lib64/libc.so.6
#6  0x00007fe27232aa78 in abort () from /lib64/libc.so.6
#7  0x00007fe272c9988a in __gnu_cxx::__verbose_terminate_handler () at ../../../../libstdc++-v3/libsupc++/vterminate.cc:95
#8  0x00007fe272ca52fa in __cxxabiv1::__terminate (handler=<optimized out>) at ../../../../libstdc++-v3/libsupc++/eh_terminate.cc:48
#9  0x00007fe272ca4359 in __cxa_call_terminate (ue_header=0x7fe24e4603e0) at ../../../../libstdc++-v3/libsupc++/eh_call.cc:54
#10 0x00007fe272ca4a87 in __cxxabiv1::__gxx_personality_v0 (version=<optimized out>, actions=6, exception_class=5138137972254386944, ue_header=0x7fe24e4603e0, context=<optimized out>) at ../../../../libstdc++-v3/libsupc++/eh_personality.cc:685
#11 0x00007fe2728ef184 in _Unwind_RaiseException_Phase2 (exc=0x7fe24e4603e0, context=0x7ffdb81dede0, frames_p=0x7ffdb81dece8) at ../../../libgcc/unwind.inc:64
#12 0x00007fe2728efbbe in _Unwind_Resume (exc=0x7fe24e4603e0) at ../../../libgcc/unwind.inc:241
#13 0x00007fe2504dd015 in std::_Sp_counted_deleter<edm::one::EDAnalyzerBase*, std::default_delete<edm::one::EDAnalyzerBase>, std::allocator<void>, (__gnu_cxx::_Lock_policy)2>::_M_dispose() [clone .cold] () from /home/smorovic/test7/CMSSW_13_1_0_pre1/lib/slc7_amd64_gcc11/pluginStreamThingAnalyzer.so
#14 0x00007fe2504dd42a in std::_Sp_counted_ptr_inplace<edm::maker::ModuleHolderT<edm::one::EDAnalyzerBase>, std::allocator<edm::maker::ModuleHolderT<edm::one::EDAnalyzerBase> >, (__gnu_cxx::_Lock_policy)2>::_M_dispose() () from /home/smorovic/test7/CMSSW_13_1_0_pre1/lib/slc7_amd64_gcc11/pluginStreamThingAnalyzer.so
#15 0x00007fe274ca5bfa in std::_Sp_counted_base<(__gnu_cxx::_Lock_policy)2>::_M_release() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_13_1_0_pre1/lib/slc7_amd64_gcc11/libFWCoreFramework.so
#16 0x00007fe274d748a2 in std::_Sp_counted_ptr_inplace<edm::ModuleRegistry, std::allocator<edm::ModuleRegistry>, (__gnu_cxx::_Lock_policy)2>::_M_dispose() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_13_1_0_pre1/lib/slc7_amd64_gcc11/libFWCoreFramework.so
#17 0x00007fe274ca5bfa in std::_Sp_counted_base<(__gnu_cxx::_Lock_policy)2>::_M_release() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_13_1_0_pre1/lib/slc7_amd64_gcc11/libFWCoreFramework.so
#18 0x00007fe274cce6a6 in edm::Schedule::~Schedule() [clone .lto_priv.0] () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_13_1_0_pre1/lib/slc7_amd64_gcc11/libFWCoreFramework.so
#19 0x00007fe274cbfba7 in edm::EventProcessor::~EventProcessor() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_13_1_0_pre1/lib/slc7_amd64_gcc11/libFWCoreFramework.so
#20 0x000000000040b731 in (anonymous namespace)::EventProcessorWithSentry::~EventProcessorWithSentry() ()
#21 0x0000000000407d99 in main ()
```